docs/updating_schemas.md

```diff
@@ -34,6 +36,7 @@ Adding a new schema
                 "id": "/profiles/{metadata object}.json",
                 "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
This should be updated to "https://json-schema.org/draft/2020-12/schema".

Reply:
changed